legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/depot/api/BaseDepotApi.java

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.depot.api;
+
+import org.apache.http.HttpStatus;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.finos.legend.sdlc.server.depot.DepotServerInfo;
+import org.finos.legend.sdlc.server.depot.auth.AuthClientInjector;
+import org.finos.legend.sdlc.server.error.LegendSDLCServerException;
+import org.finos.legend.sdlc.server.tools.StringTools;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+
+abstract class BaseDepotApi
+{
+    private static final Logger logger = LoggerFactory.getLogger(BaseDepotApi.class);
```

Comment:
Please use all caps for static fields.
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(BaseDepotApi.class);
```

Reply:
done